src/components/postIndex.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Text, View } from 'react-native'
+
+const postIndex = () => {
+  return (
+    <View>
+      <Text></Text>
+    </View>
+  )
+}
+
+export default postIndex
```

Comment:
So, the convention for React components is `PascalCase`

Reply:
Of course, we'll fix this